### PR TITLE
Update core developer setup for CHT 4.x

### LINKED ISSUE #837: Document 4.x CHT Core Developer Setup
The [CHT Core Developer Setup page](https://docs.communityhealthtoolkit.org/apps/guides/hosting/core-developer/) is out of date for 4.x.

For example, the `secure-couchdb` setup is not needed anymore in 4.x.

### PR DESCRIPTION
**Pull request by andrablaj**
Closes #837

### CODE REVIEW

File: content/en/apps/guides/hosting/core-developer.md [hunk 1]
```diff
@@ -103,6 +106,11 @@ To ensure these to exports and sourcing your rc file worked, echo the values bac
 echo $COUCH_NODE_NAME && echo $COUCH_URL
 ```
 
+#### CouchDB Setup in CHT 4.x
+Create a `docker-compose.yml` file under the `couchdb` folder. 
+
+Copy the content of the CouchDB `docker-compose` file from the of the CHT release version you are trying to run locally. For example, this is [the file](https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3A4.0.0/docker-compose%2Fcht-couchdb-clustered.yml) for the 4.0.0 release.
```
Comment (garethbowen) [inline content/en/apps/guides/hosting/core-developer.md:112]: As discussed on the call, let's link to `master` so the developer gets the latest image when setting up.

```suggestion
Copy the content of the CouchDB `docker-compose` file from the of the CHT release version you are trying to run locally. For example, this is [the file](https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose%2Fcht-couchdb-clustered.yml) for the most recent build.
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Great work, thanks.

A few suggestions inline, and it may be worth investigating whether we should run the 3.x development against a 4.x couchdb image so it does the securing automatically. I think this would greatly simplify things but obviously will need some testing!

File: content/en/apps/guides/hosting/core-developer.md [hunk 2]
```diff
@@ -118,13 +126,26 @@ Install dependencies and perform other setup tasks via an `npm` command. Note th
...
 curl -X PUT "http://medic:password@localhost:5984/_node/$COUCH_NODE_NAME/_config/httpd/WWW-Authenticate" -d '"Basic realm=\"administrator\""' -H "Content-Type: application/json"
 ```
 
+If setting up CHT 4.x, you need to startup CouchDB:
+
+```shell
+cd couchdb 
+docker-compose up
+```
+
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply content/en/apps/guides/hosting/core-developer.md]: Good idea, thanks!

**Review by garethbowen — APPROVED** [review]
Looks great, thanks for taking the time!

**Pull request merged**